test/test_shared_library.cpp

```diff
@@ -64,3 +64,9 @@ TEST(test_shared_library, failed_test) {
     FAIL();
   }
 }
+
+TEST(test_get_platform_library_name, failed_test) {
+  // create a string bigger than the internal buffer
+  std::string str(2000, 'A');
+  EXPECT_THROW(rcpputils::get_platform_library_name(str), std::runtime_error);
```

Comment:
This now checks for a throw, but the documentation still says it returns an empty string...